specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/JitPolicies.json

```diff
@@ -0,0 +1,319 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Security Center - JitNetworkAccessPolicies API - Public Preview",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2015-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "description": "Azure Active Directory OAuth2 Flow",
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/providers/Microsoft.Security/jitPolicies": {
```

Comment:
does this not work for nested resource types? in the current api signature format, it wont work for MIcrosoft.SQL/servers/server1/databases/db1

Reply:
see other comment about this